eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1524,7 +1524,7 @@ public function deleteVersion(APIVersionInfo $versionInfo)
             2
         );
 
-        if (count($versionList) === 1) {
+        if (count($versionList) === 1 && !$versionInfo->isDraft()) {
```

Comment:
Hmm, I was about to say it's actually the content state we need to check here, but if we only have one, then we can in extension assume/hope that status of content is in sync with the remaining version.

Reply:
So eave this as is?

Reply:
yeah, I think so.

But I'd like to get second opinion on this from among others @alongosz

Reply:
Current change makes sense to me.